src/processTargets/modifiers/scopeHandlers/TokenScopeHandler.ts

```diff
@@ -23,4 +24,40 @@ export default class TokenScopeHandler extends NestedScopeHandler {
         }),
     }));
   }
+
+  isPreferredOver(
```

Comment:
Maybe we should simplify this one. 
* We know that you can't have two adjacent identifiers. That means that we basically only have to check if the left one is an identifier. 
* I'm not sure if I care about length to be honest
* Returning false or undefined will give the same result. I would always return a boolean.
What about:
```js
const textA = document.getText(scopeA.domain);
return identifierMatcher.test(textA);
```

Reply:
nice